Model/Customer.php

```diff
@@ -128,6 +128,20 @@ public function addCard($cardToken)
         return $this;
     }
 
+    /**
+     * Delete card
+     *
+     * @param  string $cardToken
+     *
+     *
+     * @see    https://github.com/omise/omise-php/blob/master/lib/omise/OmiseCardList.php
+     */
+    public function deleteCard($cardToken)
+    {
+        $card = $this->customerAPI->cards()->retrieve($cardToken);
+        $card->destroy();
+    }
```

Comment:
@jacstn wonder if we should really retrieve a card here before delete.
Why not let the controller retrieve a card first then leave the delete method as just to delete?

Reply:
I don't understand what you want..